cli/src/debugger.rs

```diff
@@ -634,15 +634,15 @@ struct HaltedState {
     program_counter: u32,
     current_frame: usize,
     frame_indices: Vec<i64>,
-    stack_frames: Vec<probe_rs::debug::StackFrame>,
```

Comment:
I'd just import that at the top instead of writing this in full everytime :)

Reply:
Done :)